### ISSUE TITLE
Does it make sense to support 6.0

### ISSUE THREAD

**Issue by EllisV** [EllisV/oxid-console#44]
Does it make sense that we support OXID 6.0 as they provide having your own scripts capability?

/cc @keywan-ghadami-oxid @ddpkts

**Comment by SvenBrunk** [comment]
@EllisV I am not sure. It might make sense to migrate them all to composer scripts :)

**Comment by ddpkts** [comment]
Not sure what scripts come with 6.0, but I am still a happy console user in 5.x.x.
I might expecting to have oxid console after migrating to 6.0

**Comment by keywan-ghadami-oxid** [comment]
Hi there is a oxid 6 compatible port available 
https://github.com/OXIDprojects/oxid-console
that can be installed with composer.

**Comment by ddpkts** [comment]
@keywan-ghadami-oxid I see you did a great work porting OXID Console to V6, but I think it does not make sense use oxConsoleCommand and other custom stuff while you simply can use e.g. `symfony/console` component.

**Comment by keywan-ghadami-oxid** [comment]
@ddpkts yes for sure, I did only minor part of the work anyway I think it is exactly like that isn't it?
